creator-node/src/services/stateMachineManager/stateMonitoring/findSyncRequests.jobProcessor.js

```diff
@@ -131,7 +131,7 @@ module.exports = async function ({
     duplicateSyncReqs,
     errors,
     jobsToEnqueue: syncReqsToEnqueue?.length
-      ? { [QUEUE_NAMES.STATE_RECONCILIATION]: syncReqsToEnqueue }
+      ? { [QUEUE_NAMES.RECURRING_SYNC]: syncReqsToEnqueue }
```

Comment:
how do you know this is RECURRING vs manual

Reply:
because this is in the find-sync-requests job processor so by definition all of those jobs are going to be recurring

Reply:
oh right whoops, i misread this as `issueSyncRequest`